src/components/EventsComponents/EventsListEmpty.js

```diff
@@ -0,0 +1,30 @@
+import { Image, Center, Heading } from '@chakra-ui/react'
+import noDataImgPath from './res/img_no_data.png'
+
+const EventsListEmpty = () => {
+	return (
+		<Center
+			style={{
+				display: 'flex',
+				flexDirection: 'column',
+				alignItems: 'center',
+				marginTop: '2em'
+			}}
+		>
+			<Image
+				boxSize={{
+					base: '10em',
+					sm: '12em',
+					md: '13em',
+					lg: '14em',
+					xl: '15em',
+					'2xl': '20em',
+				}}
+				src={noDataImgPath}
+			/>
+			<Heading fontSize="10px">No events for this date.</Heading>

```

Comment:
Little detail: please translate this text to Bulgarian, since the whole app is in Bulgarian.